src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,22 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using deprecated icons
+          if (
+            updated.search(/angle-double/g) > -1 ||
+            updated.search(/bar-chart/g) > -1 ||
+            updated.search(/collapse/g) > -1 ||
+            updated.search(/line-chart/g) > -1 ||
+            updated.search(/wand/g) > -1
```

Comment:
Can we collapse this into one regex, and also include `shape="wand"` into the regex?

Reply:
Yup